modules/tensor_mechanics/src/materials/IsotropicPlasticityStressUpdate.C

```diff
@@ -42,7 +42,7 @@ IsotropicPlasticityStressUpdate::IsotropicPlasticityStressUpdate(const InputPara
     _hardening_constant(getParam<Real>("hardening_constant")),
     _hardening_function(isParamValid("hardening_function") ? &getFunction("hardening_function")
                                                            : NULL),
-    _shear_modulus(0.0),
+    // _shear_modulus(0.0),
```

Comment:
Oh my goodness yes - I completely forgot about these.  Thanks for pointing it out!